libs/vscode-ui/components/src/lib/multiple-select/multiple-select.component.ts

```diff
@@ -21,46 +21,25 @@ export class MultipleSelectComponent implements OnInit, OnDestroy {
   @Input() field: Option;
   @Input() value: string[];
   @Output() readonly valueChange = new EventEmitter<string[]>();
-  selectControl: FormControl;
-  items: string[];
-  parentFormGroup: FormGroup;
+
+  get parentFormGroup(): FormGroup {
+    return this.controlContainer.control as FormGroup;
+  }
+
+  get selectControl(): FormControl {
+    return this.parentFormGroup?.get(this.field?.name) as FormControl;
+  }
 
   constructor(private readonly controlContainer: ControlContainer) {}
 
   private readonly subscriptions = new Subscription();
 
   ngOnInit(): void {
-    this.parentFormGroup = this.controlContainer.control as FormGroup;
-    this.selectControl = this.parentFormGroup.get(
-      this.field.name
-    ) as FormControl;
     this.subscriptions.add(
       this.selectControl.valueChanges.subscribe((value) =>
         this.valueChange.emit(value)
       )
     );
-    if (this.field.items) {
```

Comment:
Another significant change I made is to not set items OnInit but instead just access them from the field.
OnInit was fine given the form is created each time a new generator or executor is opened, but it was a pain to build tests around. Now the fields update OnChange as you would expect.